.storybook/config.js

```diff
@@ -21,7 +21,11 @@ function loadStories() {
 addDecorator(story => (
   /* eslint-disable react/jsx-filename-extension */
   <>
-    <GlobalStyles fonts={Object.values(fontFaces)} />
+    <GlobalStyles
+      fonts={Object.values(fontFaces).map(fontFace => {
```

Comment:
Not super important but you could reduce the lines of code by using the compact version of the arrow function:

```
fonts={Object.values(fontFaces).map(fontFace => fontface())}
```